data/human/news.txt

```diff
@@ -2392,3 +2393,77 @@ news "graffiti on spaceport wall"
 			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinson's "Because I could not stop for Death"
 			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinson's "Because I could not stop for Death"
 			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinson's "Because I could not stop for Death"
+
+news "ccor graffiti"
+	location
+		planet "Gagarin"
+	name
+		word
+			"Graffiti painted on a wall"
+	message
+		word
+			"Someone has stenciled the emblem of the Confederated Councils of the Outer Rim here."
+			"A crest of the local government here has been defaced with a generous amount of spray paint."
+			"The insignia of a local guerrilla group has been painted on the wall here."
+		word
+			" "
+		phrase
+			"ccor crest graffiti"
+	message
+		word
+			"There's a spray-painted mess of competing emblems and slogans here. You can't make out the details, but it appears as if the burgundy markings are in support of the CCOR."
+
+phrase "ccor crest graffiti"
+	word
+		`` 2
+		`Underneath are the words "`
+	word
+		`BELENOS SHINES ACROSS THE GALAXY`
+		`THE CONFEDERATED COUNCILS WILL RETURN`
+		`RESIST THE FOREIGN OCCUPIERS`
+		`THE CCOR WILL RISE AGAIN`
+		`LIBERATE THE OUTER RIM`
+		`DISMANTLE THE SYNDICATE OLIGARCHY`
+	word
+		`!"`
+		`."`
```

Comment:
Is the idea that two thirds of the time, this should say nothing, or two thirds of the time, the introduction to the text should be skipped?
As it is, this will produce something like:
```
BELENOS SHINES ACROSS THE GALAXY."
```
and:
```
Underneath are the words "BELENOS SHINES ACROSS THE GALAXY."
```
The first is obviously not correct.

I note that there are several other phrases that do what you had this doing previously, so I guess it's fine to leave this that way since we're not changing those ones in this PR.

Reply:
Thanks for noticing that.
```suggestion
phrase "ccor crest graffiti"
	word
		`"` 2
		`Underneath are the words "`
	word
		`BELENOS SHINES ACROSS THE GALAXY`
		`THE CONFEDERATED COUNCILS WILL RETURN`
		`RESIST THE FOREIGN OCCUPIERS`
		`THE CCOR WILL RISE AGAIN`
		`LIBERATE THE OUTER RIM`
		`DISMANTLE THE SYNDICATE OLIGARCHY`
	word
		`!"`
		`."`
```